static/styles.css

```diff
@@ -3,20 +3,42 @@ html, body {
     margin: 0;
     overflow: hidden;
 }
-div.menuPanel {
-    width: 450px;
-    float: right;
+div.seedPanel {
+    position: fixed;
+    top: 45px;
+    right: 0;
+    width: 440px;
+    height: calc(100% - 45px);
+    overflow: scroll;
+    font-weight: bold;
 }
+div.seedPanel:after {
+    content : "";
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-image: url("/static/images/hardwood_bg.jpg");
+    background-size: contain;
+    width: 100%;
+    height: 100%;
+    opacity : 0.5;
+    z-index: -1;
+}
+
 div.gamePanel {
+    position: relative;
+    top: 45px;
     overflow: scroll;
-    height: 100%;
+    height: calc(100% - 45px);
```

Comment:
`calc` isn't a well-supported part of css yet, but it's close.  You typically don't want to use features that will break for some portion of your user base.

What you're trying to accomplish here can be done with just boring css (the classic way is negative margins on the top item to account for the bottom) but if you want newfangled, give flexbox a try:  flex-direction: column;